lib/lte_link_control/lte_lc_helpers.c

```diff
@@ -1561,6 +1559,80 @@ int parse_mdmev(const char *at_response, enum lte_lc_modem_evt *modem_evt)
 	return -ENODATA;
 }
 
+int parse_rai(const char *at_response, struct lte_lc_rai_cfg *rai_cfg)
+{
+	struct at_param_list resp_list;
+	int err;
+	int tmp_int;
```

Comment:
Is there a better name for this variable?

Reply:
I will combine this and enabled as they are temporary variables. This naming is used elsewhere in the file for temporary variable and I guess it's a matter of opinion whether it's ok or really bad. I don't know if `value` etc would be any better so please suggest.